packages/playground/src/main.ts

```diff
@@ -1,7 +1,13 @@
+import ui from '@nuxt/ui/vue-plugin';
 import { createApp } from 'vue';
-
+import { createRouter, createWebHistory } from 'vue-router';
 import App from './App.vue';
+import Playground from './Playground.vue';
+import './assets/main.css';
 
-import './index.css';
-
-createApp(App).mount('#app');
+const router = createRouter({ history: createWebHistory(), routes: [{ path: '/', component: Playground }] });
+const app = createApp(App);
+app.use(router);
+app.use(ui);
+document.documentElement.classList.remove('dark');
```

Comment:
@AmmarKaraja We should still properly resolve this color mode stuff by using https://vueuse.org/core/useColorMode/#usecolormode